packages/zowe-explorer-api/src/profiles/CoreUtils.ts

```diff
@@ -1,60 +0,0 @@
-/*
- * This program and the accompanying materials are made available under the terms of the *
- * Eclipse Public License v2.0 which accompanies this distribution, and is available at *
- * https://www.eclipse.org/legal/epl-v20.html                                      *
- *                                                                                 *
- * SPDX-License-Identifier: EPL-2.0                                                *
- *                                                                                 *
- * Copyright Contributors to the Zowe Project.                                     *
- *                                                                                 *
- */
-
-import { getZoweDir } from "@zowe/cli";
-import * as fs from "fs";
-import * as path from "path";
-import * as vscode from "vscode";
-
-declare const __webpack_require__: typeof require;
-declare const __non_webpack_require__: typeof require;
-
-/**
- * function to check if imperative.json contains
- * information about security or not and then
- * Imports the neccesary security modules
- */
-export function getSecurityModules(moduleName: string, isTheia: boolean): NodeRequire | undefined {
```

Comment:
isn't this considered a breaking change to the `@zowe/zowe-explorer-api`?
(my apologies if this was already discussed during a meeting and I missed it) 😢

Reply:
@JillieBeanSim

Reply:
Normally, yes, this would be a breaking change. However, this particular file/function was added as part of the 1.15.0 release, after which we realized that the `vscode` import makes the profiles API unusable outside of the VS Code context. Thus, a driving motivator for the 1.15.1 hotfix was to get this code out of the API's `profiles` folder before any extenders would break as a result of trying to adopt it.

Thanks @zFernand0 for catching this with your sharp eyes, though!